app/cloud.go

```diff
@@ -4,14 +4,107 @@
 package app
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"strings"
 	"time"
 
+	"github.com/mattermost/mattermost-server/v6/app/request"
 	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/shared/i18n"
 	"github.com/mattermost/mattermost-server/v6/shared/mlog"
 )
 
+func (a *App) NotifySystemAdminsToUpgrade(c *request.Context, currentUserTeamID string) *model.AppError {
+	userId := c.Session().Id
+
+	prefId := strings.ReplaceAll(model.CloudNotifyAdminInfo, "_", "") + "123456"
```

Comment:
Nit: Could we rename the fake user id suffix `123456` to something more descriptive like `fakeid`?